tests/FoodStorage/test_foodStorage.py

```diff
@@ -1,14 +1,19 @@
-import unittest
+from src.Constants.constants import *
 from src.FoodStorage.foodStorage import FoodStorage
 from tests.Utilities.constants import *
 from tests.Utilities.utils import *
 
+import io
+import sys
```

Comment:
How is `io` and `sys` being used in this test?

Reply:
We use it to grab the print output. I wanted to test that our print statements were displaying correctly for different cases.